### PR TITLE
remove reacquirable

### LINKED ISSUE #4568: is:reacquirable is almost exactly wrong about armor 2.0 exotics
`is:reacquirable` should check against collections'
• armor element
• base stats

### LINKED ISSUE #5116: Bug- is:reacquirable wrong about year 1 Eververse armor & masterworked armor
I ran a search for "is:reacquirable"

Some armor that should be highlighted is not:
image

The following armor should be:
image

I believe the armor that is not being highlighted is either:
1) Eververse Year 1 armor (Optimacy sets, Omega Mechanos sets, Mihaylova's, Quiao's, etc.)
2) Masterworked (Heiro Camo set, etc.)

### PR DESCRIPTION
**Pull request by delphiactual**
Closes #4568
Closes #5116 

We need better logic before allowing end users to expect consistent results.

### CODE REVIEW

File: docs/CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 ## Next
 
+* Removed `is:reacquireable` as it is untenable in its current state.
```
Comment (bhollis) [inline docs/CHANGELOG.md:3]: Maybe say "inaccurate"

**Comment by joe7dust** [comment]
So does reacquireable identify collection roll exotic armor now?

**Comment by delphiactual** [comment]
reacquirable is no longer available until it can be made consistent.

**Comment by Revadike** [comment]
It's been two years. Any chance it can be readded again?

**Comment by delphiactual** [comment]
the heuristics of reacquirable is the problem... what do you consider reacquirable?

**Comment by Revadike** [comment]
> the heuristics of reacquirable is the problem... what do you consider reacquirable?

To me It's anything you can reacquire from your collection in-game. What else could it be?

**Comment by delphiactual** [comment]
Does it have to be the exact same?

**Comment by Revadike** [comment]
> Does it have to be the exact same?

In what context? Can you give an example?

**Comment by delphiactual** [comment]
exact same roll etc

**Comment by Revadike** [comment]
> exact same roll etc

no

**Comment by delphiactual** [comment]
`-is:randomroll` rolls in collections are not random

**Comment by Revadike** [comment]
im so confused

**Comment by delphiactual** [comment]
rolls in collections are static... you cannot reacquire "Random Rolls" every exotic you pull from collections will always have the same stats, and every gun will have the same perks... Armor will have a low stat roll... for what you are asking just use

`-is:randomroll` that is the stuff you can pull from collections...

**Pull request merged**